### ISSUE TITLE
Kursnamen sind in SchILD auf 20 Zeichen begrenzt

### ISSUE THREAD

**Issue by stbaeumer** [SVWS-NRW/Schild3-BetaTest#1251]
Kursbezeichnungen, die automatisch von Untis erzeugt werden, sind (am Berufskolleg) u.U. länger. Ein Import über Kurse.dat ist dann nicht möglich.

Beispiel eines Untis-Kurses, in dem 3 Klassen im Fach Niederländisch mit der Lehrkraft ABC gekoppelt sind: NG1_GG23AGT23AGW23A_ABC

Vorschlag: Anzahl der zulässigen Zeichen auf 30 oder 40 erhöhen.

**Comment by kroerig** [comment]
Hat Untis eine Zeichenbegrenzung? Dann sollte die im SVWS gleich sein.

**Comment by stbaeumer** [comment]
max. 60 Zeichen bei Untis

**Comment by FPfotenhauer** [comment]
Die Kursbezeichnung (die in der Datenbank KurzBez heißt) auf 60 Zeichen aufzubohren würde uns im Client alleine bei der Darstellung vor Probleme stellen.

Wir haben die Kurzbezeichnung schon erhöht. Es wäre aus unserer Sicht sinvoller, wenn sich Untis hier einschränken würde.

**Comment by kroerig** [comment]
Hat Untis eine Zeichenbegrenzung? Dann sollte die im SVWS gleich sein.

**Comment by ws46** [comment]
> Dann scheinen die Österreicher hier wohl Anwendungsfälle zu haben, wo diese Länge erforderlich ist.

Untis kennt keine Kurse oder Kopplungen, deshalb werden die am Unterricht beteiligten Klassen und Lehrer in der Bezeichnung untergebracht. Wer Details wissen möchte, muss das aufdröseln.

**Comment by kroerig** [comment]
> > Dann scheinen die Österreicher hier wohl Anwendungsfälle zu haben, wo diese Länge erforderlich ist.
> 
> Wenn ich mich richtig erinnere, kennt Untis keine Kurse oder Kopplungen als eigene Strukturen, deshalb werden die am Unterricht beteiligten Klassen und Lehrer in der Bezeichnung untergebracht. Wer Details wissen möchte, muss das aufdröseln. 

Ja, die arbeiten da mit diesem Hilfskonstruckt aus alten Zeiten. Das ist histerisch gewachsen und ich denke nicht, dass sie das vor einem kompletten Redesign ändern werden, denn das würde bestehende Stundenpläne zerstören.

Da bleibt wohl nur, dass das Empfängersystem sich anpasst. Ggf. mit einer Warnung beim Import, dass die Daten nicht vollständig dargestellt werden können.

**Comment by AnneSchueller** [comment]
Ich schließe das Issue hier im Schild3-Repository.
